lab-will/model/user.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const crypto = require('crypto');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const mongoose = require('mongoose');
+const createError = require('http-errors');
+const Promise = require('bluebird');
+const debug = require('debug')('cfgram:user');
```

Comment:
The name you are using here is cfgram.  But in your package.json you are telling debug to look for user.